### DISCUSSION TITLE
How many incremental backups make sense

### DISCUSSION THREAD

**Discussion by slu-010101** [abbbi/virtnbdbackup#183] [General]
As far as I understand incremental backups do not have any impact
to the VM or the performance, since there only timestamps saved, right?

So one full backup and the rest of the month incremental backups are no problem?

Our VM's between 300 GB and 1,5 TB.

**Comment by abbbi** [comment]
Depends .. I would Go with a weekly full backup. Interesting would be how big the incremental Backups are

**Reply by slu-010101** [reply]
We started with weekly (full)backups, but this cost to much space:
VM 1,5 TB with incremental backup (daily) of ~ 25GB
VM 1,6 TB with incremental backup (daily) of ~ 12GB

That's why I'm thinking about what would make sense...

**Reply by abbbi** [reply]
Maybe consider backing up to a filesystem that supports deduplication, this should allow for quite some space to be saved and allows to keep more full backups.
